src/routes.js

```diff
@@ -118,7 +118,17 @@ export default [
     }
   },
   {
-    path: `/create`,
+    path: `/selectnetwork`,
```

Comment:
```suggestion
    path: `/network`,
```
won't this resolve to `/session/network`?

Reply:
? Why would it? I mean, `/session/network` doesn't even exist as a route right?

Reply:
ah my mistake. i thought theses were child routes...